content/en/docs/performance/resource_consumption.md

```diff
@@ -0,0 +1,122 @@
+---
+draft: false
+linktitle: Kmesh resource consumption
+menu:
+  docs:
+    parent: performance
+    weight: 2
+title: Kmesh resource consumption
+toc: true
+type: docs
+---
+# 1. Environment setup
+![resource_env](images/resource_test_env.png)
+
+- K8S version: v1.27
+- Kmesh version: 0.4
+- Kernel version: 5.10
+- Node flavor: 8U16G
+- testing tool: fortio
+- metric collecting tools: top, bpftop
+
+
+# 2. Test case 1 - POD with CPU limit scenario，collect Kmesh eBPF CPU usage
+## 2.1 Deploy 1 App A and set App A POD's CPU limit，then generage load and collect Kmesh eBPF CPU usage
+![resource_test1](images/resource_test1.png)
+
+Set the CPU limit for App A to 1 (1 CPU), and collect corresponding Kmesh eBPF CPU consumption.
+
+**Note: With the system having 8 cores and a CPU limit of 1, it means the POD can consume up to 12.5% of the CPU.**
+
+Testing result：
+![resource_result1](images/resource_test_result1.png)
+
+**Results and conclusions: When APP A uses up 1 CPU, the user space occupies 4%, the kernel space occupies 7.4%, totaling 11.4% of CPU consumption, which is less than the POD CPU limit of 12.5%. There are two possible reasons.**
+
+- APP A and Kmesh eBPF share the POD CPU limit, with Kmesh eBPF CPU being restricted by the POD CPU limit.
```

Comment:
why ebpf share pod cpu usage with pod?  ebpf is run in kernel, whild pod cpu usage includes all the process cpu usage within the pod

Reply:
we have presented and discussed this testing result in the weekly meeting with @nlgwcy , and everybody are aware of this.